packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/web_resource_response.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// Contains information about the response for the request.
+///
+/// Platform specific implementations can add additional fields by extending
+/// this class.
+///
+/// {@tool sample}
```

Comment:
nit: this can be removed. It actually doesn't do anything on `pub.dev` or within our plugin.